applications/SwimmingDEMApplication/swimming_DEM_application.cpp

```diff
@@ -77,7 +119,7 @@ void KratosSwimmingDEMApplication::Register()
   KRATOS_REGISTER_ELEMENT("MonolithicDEMCoupledWeak2D", mMonolithicDEMCoupledWeak2D)
   KRATOS_REGISTER_ELEMENT("MonolithicDEMCoupledWeak3D", mMonolithicDEMCoupledWeak3D)
   KRATOS_REGISTER_ELEMENT("RigidShellElement", mRigidShellElement)
-  KRATOS_REGISTER_ELEMENT("SphericSwimmingParticle3D", mSphericSwimmingParticle3D)
+  KRATOS_REGISTER_ELEMENT("SwimmingParticle3D", mSwimmingParticle3D)
```

Comment:
This name change will require updating F-DEMpack, we could leave it with the old name or update the GUI...

Reply:
Actually, I want to restore the name to the old one, since it is more correct. I changed everything after changing the particle's name, but when the template is applied to the SphericParticle it should be a spheric_swimming_particle. Let me do it.